python/lsst/obs/base/tests.py

```diff
@@ -70,13 +70,12 @@ def setUp_tests(self, butler, mapper, dataIds):
 
         Parameters
         ----------
-        butler: lsst.daf.persistence.Butler
+        butler: `lsst.daf.butler.Butler`
             A butler object, instantiated on the testdata repository for the
             obs package being tested.
-        mapper: lsst.obs.CameraMapper
-            A CameraMapper object for your camera, instantiated on the testdata
-            repository the obs package being tested.
-        dataIds: dict
+        mapper: `None`
+            A Gen2 parameter that should now always be `None`.
```

Comment:
Why is this still here?  I guess some compatibility issue?  Is this something that could/should be deprecated for removal?